Changes based on feedback on last pull request

rogermeadows opened the pull request:
I think this pull request includes all of the changes included in comments and discussion regarding the last pull request. 
This pull request has been cleaned up and is only one commit and should be fast forward so rebase merge can be used. Or squash merge may accomplish the same thing.

rogermeadows committed:
Changes based on feedback on last pull request

SunyataZero commented on fork-clone-workflow.md:
```diff
@@ -49,13 +49,14 @@ cd /opt/repos
+[github](git-glossary.md#github) account with your public SSH key. See [Cloning using SSH](cloning_using_ssh.md) for 
+more information on using the `Clone with SSH` option to access remote repositories.
+As an alternative to SSH, you may select `Use HTTPS` to access the `Clone with HTTPS` URL. 
+The HTTPS option uses an HTTPS URL and will require you to supply your [github](git-glossary.md#github)
```
"and will require you to supply your github account password each time you access your fork" -- Do we want to add here that there is a way to store the password?

rogermeadows replied:
I think that will be operating system specific. Perhaps we can say that the operating system you are using may provide a tool that will supply the password for you? I do not have experience using a tool like that for this situation. I almost always use SSH.

SunyataZero replied:
Okay, this sounds good to me too. I first thought that there was a shared way of doing this since there is a git command related to this, but looking closer it seems it's more platform specific

SunyataZero commented on fork-clone-workflow.md:
```diff
@@ -49,13 +49,14 @@ cd /opt/repos
-The HTTPS option uses a different URL and will require you to supply your [github](git-glossary.md#github)
-account password each time you access your fork. The SSH option does not require you to supply your [gihub](git-glossary.md#github)
-when accessing your fork.
+`Clone with HTTPS`. There is also a `Clone with SSH` option which requires that you have confiured configured your
```
> There is also a `Clone with SSH` option which requires that you have confiured configured your github account with your public SSH key. See Cloning using SSH for more information on using the `Clone with SSH` option to access remote repositories.

Can we move this to the end of the section/paragraph?

rogermeadows replied:
Yes. I will update the branch in my fork that is referenced in the pull request. I think that will update the pull request.

SunyataZero commented on fork-clone-workflow.md:
```diff
@@ -64,30 +65,30 @@ git-info
-[references](git_glossary.md#references) to remote repositories. 
-git automatically creates the `origin` reference when the clone is created. The `git remote -v` command lists the known
-[references](git_glossary.md#references) and the values associated with each reference.
+[remote](git_glossary.md#references) names that point to remote repositories. 
```
For "`[remote](git_glossary.md#references)`" "`[remotes](git_glossary.md#references)`" can we change from "#references" to "#remote"?

rogermeadows replied:
Yes. I missed those two when changing 'reference(s)' to 'remote)s)'/

SunyataZero commented on git-glossary.md:
```diff
@@ -59,10 +59,12 @@ and if approved will be merged into the organization repository.
+## remote
+A string name that points to a repository. The value associated with a remote is the URL used to access the remote repository.
+A remote serves as a short alias for a longer URL. 
+For the [fork/clone development workflow](fork-clone-workflow.md) a repository will be in one of three locations. 
```
> a repository will be in one of three locations.

Can we change the list below this line to start with the locations and then describe how to access it? (Alternatively we could use another text instead of the above)

rogermeadows committed:
Changes based on pull request comments

One pull request comment is still being discussed for the fork/clone workflow document.

rogermeadows committed:
Changes based on pull request comments
